config/analytics_hidden_pii.yml

```diff
@@ -8,3 +8,6 @@
   :emails:
   - to
   - personalisation
+  :users:
+  - id
+  - full_name
```

Comment:
just to confirm it's ok to send this under hidden pii?

Reply:
Yeah, it'll be encrypted and masked in BQ and only visible to people with special privileges.